File: backend/Tasque.Core.Common/Entities/Notifications/TaskMovedNotification.cs
using System;
using System.Collections.Generic;
using System.Linq;
namespace Tasque.Core.Common.Entities.Notifications
{
    public class TaskMovedNotification : Notification
    {
        public override NotificationType Type => NotificationType.TaskMoved;
        public override int RecieverId => TaskAuthorId;
        public int PreviousColumnId { get; set; }
        public int NewColumnId { get; set; }
        public int TaskId { get; set; }
        public int MovedById { get; set; }
        public int TaskAuthorId { get; set; }
    }
}

File: frontend/src/core/models/notifications/task-moved-notification.ts
import { Notification } from './notification';
import { NotificationType } from './notification-type';

export interface TaskMovedNotification extends Notification {
    type: NotificationType.TaskMoved,
    taskId: number,
    previousColumnId: number,
    newColumnId: number,
    movedById: number
}
